docs/app/pages/css/css-sections/icons/ux-icons/icon-preview/icon-preview.component.less

```diff
@@ -0,0 +1,36 @@
+@import '../../../../../../assets/styles/variables.less';
+
+.btn-text {
```

Comment:
Could this be something like `.uxd-icon-button` or something? Bit more descriptive. `uxd` is the prefix that we use for stuff that only applies to the documentation, so it won't collide with anything else.

Reply:
updated